javalib/src/main/scala/java/lang/Long.scala

```diff
@@ -226,8 +226,8 @@ object Long {
     approxQuot.toString(radix) + paddingZeros.jsSubstring(remStr.length) + remStr
   }
 
-  private def parseLongFail(s: String): Nothing =
-    throw new NumberFormatException(s"""For input string: "$s"""")
+  private[lang] def parseLongFail(s: String): Nothing =
```

Comment:
```suggestion
  private def parseLongFail(s: String): Nothing =
```

I don't think this PR introduces any usage anymore now.